start/pom.xml

```diff
@@ -161,126 +84,52 @@
-                    <!-- tag::loose-app[] -->
-                    <looseApplication>true</looseApplication>
-                    <!-- end::loose-app[] -->
-                    <packageFile>${package.file}</packageFile>
-                    <include>${packaging.type}</include>
-                    <bootstrapProperties>
-                        <default.http.port>${http.port}</default.http.port>
-                        <default.https.port>${https.port}</default.https.port>
-                        <app.context.root>${app.name}</app.context.root>
-                    </bootstrapProperties>
-                </configuration>
+                <version>${version.liberty-plugin}</version>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
+                <version>${version.maven-war-plugin}</version>
             </plugin>
-            <!-- Plugin to run unit tests -->
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>${version.maven-surefire-plugin}</version>
-                <executions>
-                    <execution>
-                        <phase>test</phase>
-                        <id>default-test</id>
-                        <configuration>
-                            <excludes>
-                                <exclude>**/it/**</exclude>
-                            </excludes>
-                            <reportsDirectory>
-                                ${project.build.directory}/test-reports/unit
-                            </reportsDirectory>
-                        </configuration>
-                    </execution>
-                </executions>
-                <configuration>
-                    <skipTests>${skipTests}</skipTests>
-                </configuration>
             </plugin>
             <!-- Plugin to run functional tests -->
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-failsafe-plugin</artifactId>
                 <version>${version.maven-failsafe-plugin}</version>
-                <executions>
-                    <execution>
-                        <phase>integration-test</phase>
-                        <id>integration-test</id>
-                        <goals>
-                            <goal>integration-test</goal>
-                        </goals>
-                        <configuration>
-                            <includes>
-                                <include>**/it/**</include>
-                            </includes>
-                            <systemPropertyVariables>
-                                <liberty.test.port>${http.port}</liberty.test.port>
-                                <war.name>${app.name}</war.name>
-                            </systemPropertyVariables>
```

Comment:
After removing this how were you thinking the port gets into the client ITs (doing `String port = System.getProperty("liberty.test.port");` )

Reply:
`System.getProperty("liberty.test.port", "9080");` for the default, which always works in my cases and, if necessary, override it on executing the Maven ITs with `-Dliberty.test.port=9081` ...

Reply:
What I really meant to say is that the previous approach parameterized the client ITs while parameterizing the server at the same time, via this config:

                    <bootstrapProperties>
                        <default.http.port>${http.port}</default.http.port>
                        <default.https.port>${https.port}</default.https.port>
                        <app.context.root>${app.name}</app.context.root>
                    </bootstrapProperties>

If you passed a 9081 value for `liberty.test.port` how would the server pick up the configuration to launch ?

Reply:
I see your point. This config was actually something I'd remove from the getting-started-guide at least, since it overloads the user with something that is IMO too detailed for that abstraction level already.

So, I'd keep the basic sample lean, and maybe only point to the advanced configuration, maybe to a different guide or docs. How we could make it work in a more leaner way without touching the pom.xml is to define the server.xml with `<httpEndpoint httpPort="${default.http.port}" ... />`, and override the var via the bootstrap properties when running Maven: `mvn ... -DbootstrapProperties=default.http.port=9081`.

But, I'd actually keep it out of the first guide to not confuse. For getting started users it's IMO better to start with a lean config, since it's not required to run Liberty. Otherwise, developers might assume that all that config is required to get started, which is not true for our lean technology.